docs-markdown/markdownlint-custom-rules/image.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+
```

Comment:
@bharney-msft  - Is this file just a stub? It is called image.js but it has xref rules?

Reply:
I'm just asking because I own the task to write image linting rules :)

Reply:
Its just a stub that I was using for the task to mock out some sudo code.

Reply:
Got it.